lib/taskana-core/src/main/java/pro/taskana/monitor/internal/MonitorMapperSqlProvider.java

```diff
@@ -176,10 +194,16 @@ public static String getTaskIdsForSelectedItems() {
         + "</if>"
         + "</if>"
         + "<if test=\"_databaseId == 'db2'\">"
-        + "#{selectedItem.upperAgeLimit} >= (DAYS(${timestamp})"
-        + " - DAYS(CAST(#{now} as TIMESTAMP))) AND "
-        + "#{selectedItem.lowerAgeLimit} &lt;= (DAYS(${timestamp})"
-        + " - DAYS(CAST(#{now} as TIMESTAMP))) "
+        + "#{selectedItem.upperAgeLimit} >= (DAYS(${timestamp}) - DAYS(CAST(#{now} as TIMESTAMP)))"
```

Comment:
Am I stupid or is this statement `(DAYS(${timestamp}) - DAYS(CAST(#{now} as TIMESTAMP)))` exactly `AGE_IN_DAYS`? If so: why don't we just reuse that column. This would also reduce this if statement ;)

Reply:
your right, but in this query you don't have an age_in_days property